guide/dev/code/submodules.md

```diff
@@ -5,31 +5,55 @@ layout: website-normal
 
 ## Setting Up Forks
 
-If you're contributing regularly, you'll want your submodules to pull from the `apache` repo
+If you're contributing regularly, you'll want your submodules to pull from the repo in the Apache GitHub org
 and push to your fork.
-You'll need to create a fork and the set up your local submodules should then have:
+You'll need to create a fork and then set up your local submodules should then have:
 
 * your fork as the `origin` remote repo
-* the Apache github repo as the `upstream` remote repo
-* optionally the repo at the Apache git server as the `apache` remote repo  
+* the Apache GitHub repo as the `upstream` remote repo, 
+  optionally with `origin` set as the `push` target (since you can't push to `github.com/apache`)
+  and fetch set up to get pull-request branches
```

Comment:
remove this line - only committers need PR branches and the "committers guide" in the website should describe how to set that up. For the majority of potential contributors, the PR branches are just noise.